workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/CreateEvolvableTokensFlowHandler.kt

```diff
@@ -1,15 +1,17 @@
 package com.r3.corda.lib.tokens.workflows.flows.evolvable
 
 import co.paralleluniverse.fibers.Suspendable
+import com.r3.corda.lib.tokens.workflows.internal.flows.finality.ObserverAwareFinalityFlowHandler
 import net.corda.core.contracts.requireThat
-import net.corda.core.flows.*
-import net.corda.core.node.StatesToRecord
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.FlowSession
+import net.corda.core.flows.SignTransactionFlow
 import net.corda.core.transactions.SignedTransaction
 import net.corda.core.utilities.unwrap
 
-class CreateEvolvableTokensFlowHandler(val otherSession: FlowSession) : FlowLogic<SignedTransaction>() {
+class CreateEvolvableTokensFlowHandler(val otherSession: FlowSession) : FlowLogic<Unit>() {
```

Comment:
Would we maybe want to return the signed transaction? Maybe the responder might need it for subsequent things in the parent flow?

Reply:
It's because of the return type of observer aware finality flow, I will check if I could work around that in the handler.